src/Compiler/test/Microsoft.NET.Sdk.Razor.SourceGenerators.Tests/RazorSourceGeneratorComponentTests.cs

```diff
@@ -138,4 +142,153 @@ @inherits BaseComponent
         Assert.Equal(4, result.GeneratedSources.Length);
         await VerifyRazorPageMatchesBaselineAsync(compilation, "Views_Home_Index");
     }
+
+    // AddComponentParameter is not available regardless of the lang version since we reference AspNetCore v7.
```

Comment:
This is suprising to me, as it seems like we're not actually testing what customers are likely to be running on. Do we have a plan to switch over to `8.0` and invert these tests?

Reply:
Well, the version is intentionally fixed per https://github.com/dotnet/razor/pull/8435#discussion_r1140498700. I will attempt to upgrade it to 8 in this PR.

Reply:
@jjonescz i'd rather we didn't upgrade to 8 in this PR. Rather let's 

1. Keep this PR focused on the initial purpose
2. File an issue to fix this up once the uprade to .NET 8 is completed

Worried about this PR getting too much creep into it. The Net 8 problem is an existing one so let's just take an item to fix it after.